CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ option(WASMEDGE_PLUGIN_OPENCVMINI "Enable and build WasmEdge opencvmini plugin."
 option(WASMEDGE_PLUGIN_PROCESS "Enable and build WasmEdge process plugin." OFF)
 #   WasmEdge plug-in: Stable-diffusion.
 option(WASMEDGE_PLUGIN_STABLEDIFFUSION "Enable and build WasmEdge stable-diffusion plugin." OFF)
-option(WASMEDGE_PLUGIN_STABLEDIFFUSION_CUBLAS "Enable CUBLAS in the stable-diffusion plugin." OFF)
+option(WASMEDGE_PLUGIN_STABLEDIFFUSION_CUDA "Enable CUDA in the stable-diffusion plugin." OFF)
```

Comment:
What's the reason for renaming this option?

Reply:
Because of the stable diffusion.cpp also renamed [it](https://github.com/leejet/stable-diffusion.cpp/commit/dcf91f9e0f2cbf9da472ee2a556751ed4bab2d2a).

Reply:
I see.